.github/workflows/nightly-build.yml

```diff
@@ -34,6 +34,27 @@ jobs:
         with:
           name: natives-${{ matrix.os }}
           path: cli/target/ideasy*
+      - name: Build MSI with WixToolSet
```

Comment:
I tried to integrate the whole MSI installation into maven, but the plugins are not compatbile with the latest version 5. https://wix-maven.github.io/wix-maven-plugin/

So I chose this place to insert the Wixtoolset installation. I would also be possible to move this to the last `deploy` job, but there we are using `ubuntu` as a runner and `dotnet` would not work properly. 

I think we could also move the whole thing into a separate Github Action. But maybe, we could create a new issue for that.

If you have any suggestions, than please comment on this